WordPress/Classes/ViewRelated/QR Login/Coordinators/QRLoginScanningCoordinator.swift

```diff
@@ -0,0 +1,66 @@
+import Foundation
+
+protocol QRLoginScanningView {
+    func showError(_ message: String)
+    func showCameraLayer(_ layer: AVCaptureVideoPreviewLayer)
+}
+
+class QRLoginScanningCoordinator: NSObject {
+    let parentCoordinator: QRLoginCoordinator
+    let view: QRLoginScanningView
+
+    init(view: QRLoginScanningView, parentCoordinator: QRLoginCoordinator) {
+        self.view = view
+        self.parentCoordinator = parentCoordinator
+    }
+    // MARK: - Strings
+    private struct Strings {
```

Comment:
[nit] Wdyt of using an enum instead of a struct? Enums are pure namespaces so they can't be instantiated, which is convenient for constants

```suggestion
    private enum Strings {
```

Reply:
Nice, updated here: 27ee6ebc24a6a8079a85040df0677fc4ed27434f